packages/jazz-tools/src/coValues/coStream.ts

```diff
@@ -138,6 +195,10 @@ export class CoStream<Item = any> extends CoValueBase implements CoValue {
     return instance;
   }
 
+  /**
+   * Push items to this `CoStream`
```

Comment:
Maybe explain here how this will append it's to the current session's log?

Reply:
Added explanation and example